src/azure-cli/azure/cli/command_modules/dms/commands.py

```diff
@@ -15,9 +15,9 @@
 
 def dms_api_exception_handler(ex):
     from knack.util import CLIError
-    from azure.mgmt.datamigration.models import ApiErrorException
+    from azure.mgmt.datamigration.models import ApiError
```

Comment:
Could you tell why we need change code in `dms` module since you just need migrate `ams` module?